web/modules/asu_modules/asu_secure_superadmin/src/Services/ChangeSuperAdminService.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Drupal\asu_secure_superadmin\Services;
+
+use Drupal\cas\Service\CasUserManager;
+use Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\user\Entity\User;
+use Drupal\user\Event\AccountCancelEvent;
+use Drupal\Core\Password\DefaultPasswordGenerator;
+
+/**
+ * Change the SuperAdmin (uid 1) to a new user.
+ */
+class ChangeSuperAdminService {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * The event dispatcher.
+   *
+   * @var \Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher
+   */
+  protected $eventDispatcher;
+
+  /**
+   * The CAS user manager.
+   *
+   * @var \Drupal\cas\Service\CasUserManager
+   */
+  protected $casUserManager;
+
+  /**
+   * The password generator.
+   *
+   * @var \Drupal\Core\Password\DefaultPasswordGenerator
+   */
+  protected $passwordGenerator;
+
+  /**
+   * An array of ASU Enterprise Technology admins.
+   *
+   * @var string[]
+   */
+  public const ETADMINS = [
+    'rmlebla1',
+    'dornela3',
+    'tkaiserb',
+    'tlstarr',
+    'mmilner6',
+    'apersky',
+    'mlsamuel',
+    'cphill',
+    'ddavis35',
+    'gamille7',
+    'igardun1',
+    'dlevy4',
+    'abrockha',
+    'jmitriat',
+    'tbutterf',
+    'stwilli2',
+    'imorale2',
+    'ddoozan',
+    'kdmarks',
+    'mmilner6',
+    'mjenki10',
+    'oatkar',
```

Comment:
We don't need to include our student worker here.

Reply:
Removing.